public/lnd-child-process.js

```diff
@@ -76,6 +76,7 @@ module.exports.startLndProcess = async function({
     ]);
   } else {
     args = args.concat([
+      '--autopilot.active',
```

Comment:
In this [comment](https://github.com/lightninglabs/lightning-app/pull/1000/commits/f577a413ad1f2db3aaeaeefeadc73d23410f2e57#r266288726) you said you removed the flag, but it seems to be re-added here?

Note it will need to be removed for bos score :)